test/simulation/tests.go

```diff
@@ -21,23 +21,23 @@ import (
 // by their preceding ones.
 func testNetworkInit(net *xudtest.NetworkHarness, ht *harnessTest) {
 	for _, node := range net.ActiveNodes {
-
 		// Verify connectivity.
-		req := &xudrpc.GetInfoRequest{}
-		res, err := node.Client.GetInfo(ht.ctx, req)
-		ht.assert.NoError(err)
-		ht.assert.NotNil(res.Lnd["BTC"])
-		ht.assert.NotNil(res.Lnd["LTC"])
-		ht.assert.Len(res.Lnd["BTC"].Chains, 1)
-		ht.assert.Equal(res.Lnd["BTC"].Chains[0], "bitcoin")
-		ht.assert.Len(res.Lnd["LTC"].Chains, 1)
-		ht.assert.Equal(res.Lnd["LTC"].Chains[0], "litecoin")
-
-		// Set the node public key.
-		node.SetPubKey(res.NodePubKey)
-
-		// Add pair to the node.
-		ht.act.addPair(node, "LTC", "BTC", xudrpc.AddCurrencyRequest_LND)
+		for {
+			// retry interval
+			time.Sleep(100 * time.Millisecond)
```

Comment:
Better to start with an initial attempt, and do retries on failure. 

Also maybe limit the retries/total time below the default test timeout, specially if you increased it to 30 sec.

Reply:
Good point. I've changed the code to do the initial attempt first and added a timeout of 10 second. Also lowered the context timeout to 15 seconds.